src/DenoHTTPWorker.ts

```diff
@@ -105,10 +107,15 @@ export const newDenoHTTPWorker = async (
 
   let scriptArgs: string[];
 
+  let socketFile = `${Math.random()}-deno-http.sock`.substring(2);
```

Comment:
`crypto.randomUUID()` is probably an easier way to get a random string, and it'll give us filenames that are of a consistent length